modules/monitoring-creating-a-monitoringstack-object-for-cluster-observability-operator.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// monitoring/cluster-observability-operator/configuring-the-cluster-observability-operator-to-monitor-a-service.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-a-monitoringstack-object-for-cluster-observability-operator_{context}"]
+= Creating a MonitoringStack object for the Cluster Observability Operator
+
+To be able to discover the sample `ServiceMonitor` object you created by using the documentation in "Specifying how a service is monitored for Cluster Observability Operator", you must create and configure a `MonitoringStack` object.
+
+You can create a `MonitoringStack` object to discover the sample `ServiceMonitor` object. 
+The `ServiceMonitor` object monitors the sample service named `prometheus-coo-example-app` in the `ns1-coo` namespace.
```

Comment:
@abrennan89 You are wrong here for`ServiceMonitor` object is config data and can't monitor anything. @bburt-rh You may change line 11 and 12 as the following, line 50 has given detail how `ServiceMonitor` specify services to be monitored.
 You can create a `MonitoringStack` object to discover the target service to be monitored by using`ServiceMonitor` object and scrape data from the target.

Reply:
@lihongyan1 
I made changes that I think should address this issue. Can you please review again? Thanks.